src/components/PokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+const PokemonCard = ({pokemonName}) => {
+    const [ pokemon, setPokemon] = useState({})
```

Comment:
Als je wilt controleren of het al is ingeladen of niet, dan zou ik een default waarde van `undefined` gebruiken
```suggestion
    const [ pokemon, setPokemon] = useState();
```